nix/set-git-rev/set-git-rev.nix

```diff
@@ -0,0 +1,15 @@
+{ system, nixpkgs, haskellNix, flake-utils, ... }:
+let
+  src = ./.;
+  indexState = "2024-08-20T21:35:22Z";
```

Comment:
Or this `indexState` ? Aren't we already defining it in other nix file?

Reply:
Yes there is duplication, but this tools have nothing to do with our libraries in terms of constraints, so I'm not even trying to reuse any code, at the price of some duplication. OTOH you can extract this directory and copy it over. It's a self contained tool. `rewrite-libs` is coming as well in the same shape for the node bump. A compromise.